data/en/test_currency.json

```diff
@@ -10,26 +10,61 @@
     "messages": {
         "NUMBER_TOO_LARGE": "Number is too large",
         "NUMBER_TOO_SMALL": "Number cannot be less than zero",
-        "INVALID_NUMBER": "Please enter an integer"
+        "INVALID_DECIMAL": "Please enter a number to 2 decimal places"
```

Comment:
You should use the placeholder value `%(max)d` instead of `2` so that if the answers change the message will be updated automatically. This would also mean the answers can have varied decimal places.